### PR TITLE
User Replies and View Transaction Button

### PR DESCRIPTION
**Pull request by CUTolu2021**
### Why This PR?
To enhance user experience and improve admin functionality for the HIN Telegram bot
### What Does This PR Do?
Reformatted all user replies to improve readability and visual appeal
Added logic to the transaction button for admins, enabling it to function as intended
### Anything for the Reviewer to Take Note Of?
None

### CODE REVIEW

File: src/core/controllers/bot.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { Users, IUser } from '../models/users';
 import { Admins, IAdmin } from '../models/admins';
 import { Accounts } from '../models/accounts';
 import { Transactions } from '../models/transactions';
+import { Quarters } from '../models/quarters';
+import { connectMongoDB } from '../database';
```
Comment (Richydos2305) [inline src/core/controllers/bot.ts]: This shouldn't be here

File: src/core/controllers/bot.ts [hunk 2]
```diff
@@ -14,6 +16,8 @@ let username;
 let loggedInUser: IUser;
 let loggedInAdmin: IAdmin;
 
+connectMongoDB();
+
```
Comment (Richydos2305) [inline src/core/controllers/bot.ts]: This shouldn't be here

File: src/core/controllers/bot.ts [hunk 3]
```diff
@@ -72,7 +77,7 @@ bot.command('transactions', async (ctx) => {
   } else {
     await ctx.reply('Not an Admin. You do not have access to this command');
   }
-});
+});*/
```
Comment (Richydos2305) [inline src/core/controllers/bot.ts]: Why did you comment out this command?

**Review by Richydos2305 — CHANGES_REQUESTED** [review]

File: src/core/controllers/bot.ts (hunk 1 above)
Comment (CUTolu2021) [reply src/core/controllers/bot.ts]: My apologises

File: src/core/controllers/bot.ts (hunk 3 above)
Comment (CUTolu2021) [reply src/core/controllers/bot.ts]: cause i have it ready for the button 
i will be deleting it in the next commit

File: src/core/controllers/bot.ts (hunk 2 above)
Comment (CUTolu2021) [reply src/core/controllers/bot.ts]: More apologises

**Review by Richydos2305 — APPROVED** [review]

**Pull request merged**